wsgi/odyssy/odyssy/urls.py

```diff
@@ -30,5 +30,9 @@
     url(r'^news/', include('news.urls')),
     url(r'^tag/', include('tag.urls')),
     url(r'^announcement/', include('announcement.urls')),
+    url(r'.html/$', include('serve_static.urls')),
```

Comment:
Sir, here problem is that there are certain html files which are not included in any of the apps, yet we have to serve them. For example aboutus.html, faculty.html, director_message.html etc. 

`staticfiles` app is only for serving static contents such as css, js, images etc. And I have used the same to serve static files. For serving html files which are not linked to the any of the apps yet, I suggested Monal to create a different app, which catches all the urls ending at `.html` and display the requested `html` file which is stored  in `odyssy/wsgi/templates` folder.